src/mixer/audio.ts

```diff
@@ -270,6 +275,16 @@ export class AudioEngine extends ((EngineEmitter as unknown) as {
     return player;
   }
 
+  // Wavesurfer needs cleanup to remove the old audio mediaelements. Memory leak!
+  public destroyPlayer(number: number) {
+    const existingPlayer = this.players[number];
+    if (existingPlayer != null) {
+      // already a player setup. Clean it.
+      existingPlayer.cleanup();
+    }
+    this.players[number] = undefined;
```

Comment:
Is it null or is it undefined?

Reply:
Hmm, I did this to get around ts throwing a wobbly, != null apparently also includes undefined and worked. Having said that... it's now working... I don't know.